realm/realm-library/src/androidTest/java/io/realm/services/RemoteProcessService.java

```diff
@@ -34,6 +34,7 @@
 
 /**
  * Helper service for multi-processes support testing.
+ * @deprecated use {@link RemoteTestService} instead.
```

Comment:
It is still used in `RealmInterprocessTest` which I don't have time to refactor yet.